relays/bin-substrate/src/cli/send_message.rs

```diff
@@ -22,15 +22,33 @@ use crate::cli::{
 	TargetSigningParams,
 };
 use bp_message_dispatch::{CallOrigin, MessagePayload};
-use bp_runtime::messages::DispatchFeePayment;
 use codec::Encode;
 use frame_support::weights::Weight;
 use relay_substrate_client::{Chain, TransactionSignScheme};
 use sp_core::{Bytes, Pair};
 use sp_runtime::{traits::IdentifyAccount, AccountId32, MultiSignature, MultiSigner};
 use std::fmt::Debug;
 use structopt::StructOpt;
-use strum::VariantNames;
+use strum::{EnumString, EnumVariantNames, VariantNames};
+
+/// Relayer operating mode.
+#[derive(Debug, EnumString, EnumVariantNames, Clone, Copy, PartialEq, Eq)]
+#[strum(serialize_all = "kebab_case")]
+pub enum DispatchFeePayment {
+	/// The dispacth fee is paid at the source chain.
+	AtSourceChain,
```

Comment:
Can we rename (in `strum`?) so that the CLI options are just `source` and `target`?

Reply:
Yes, but what's the reasoning? IMO it is better to have same id everywhere than trying to remember how selected variant is called in CLI

Reply:
It's about CLI user experience. There is already a lot of options to pass and shorter terms are easier to remember than multi-word sentences. I'm okay renaming the enums too, but being more explicit in the code do add readability value IMHO.

Reply:
There are two type of users imo:
- if you're familiar with relay code, then you should have clear association between this cli option and enum and it'll be easier to you to just remember **single** variant;
- if you're not familiar with relay code, then you'll learn cli options either from examples or from clip help, where you'll find everything you need. And since you have no any initial knowledge, there'll be no shorter vs longer confusion. And I really doubt that remembering `at-source-chain` vs `source` is a problem. I'd rather argue that understanding what `--dispatch-fee-payment` means is a problem. And then seeing `--dispatch-fee-payment=at-source-chain` makes it easier to understand than seeing '--dispatch-fee-payment'=source`.